### DISCUSSION TITLE
I have a question regarding Github error

### DISCUSSION THREAD

**Discussion by 1996haseeb** [uBlockOrigin/uAssets#15135] [Q&A]
> The following error appeared when I created the project with version control in RStudio: "waitpid for git-remote-https failed: No child processes." After some digging online, it seems to be possible to run it as an administrator, however when I did that, it opened up a command prompt environment rather than allowing me to [easypose ](https://broadapk.com/easy-pose-mod-apk/) work directly from RStudio, so I wasn't sure where to go from there. Also, I installed the latest version of Git, but it didn't help. Is there anyone who can help me?

In addition, I used usethis::use_github as a way to upload my files to Github since I couldn't do it normally. Do you know if it is possible to specify use_github for private repositories as opposed to public ones? As I understand it, I can always set it to private on Github. Thank you!

**Comment by mapx-** [comment]
- not related to uAssets
- use google
`https://community.rstudio.com/t/github-error-and-use-github-question/119982/3`

**Reply by stephenhawk8054** [reply]
Looks like someone is testing some bots. I see this user having posted on uAssets' discussion a few times with different unrelated topics (deleted them already).
